brigade.js

```diff
@@ -164,8 +193,9 @@ function runCheck(e, p) {
 function runSuite(e, p) {
   if (e.revision.ref == "master" ) {
     Group.runEach([
-      checkRun(e, p, test, "Test"),
+      checkRun(e, p, testUnit(), "Unit Test"),
```

Comment:
The referenced functions in this and line 198 should just be: `testUnit` and `testCLI` respectively (no parens).